ntest_test_cases/src/lib.rs

```diff
@@ -101,6 +101,15 @@ mod syn_helper;
 ///     panic!("I am panicing");
 /// }
 /// ```
+///
+/// Test functions with a `Result` return are also supported:
+///
+/// ```
+/// #[test]
```

Comment:
This is a normal rust test without using test_case macro.

Reply:
Oops myself!